### PR TITLE
fix for nerfed slime mobs splitting (#3930)

### LINKED ISSUE #3930: Magma cube spawner nerf not work perfectly
### What behaviour is expected:
___What you expected to see___
When a magma cube is killed, it split into 2-4 smaller magma cubes. The smaller magma cubes nerfed.

### What behaviour is observed:
___What you actually saw___
When a magma cube is killed, it split into 2-4 smaller magma cubes. The smaller magma cubes don't nerfed.

### Steps/models to reproduce:
___This may include a build schematic, a video, or detailed instructions to help reconstruct the issue___
Kill magma cubes from magma cube spawner.

### Plugin list:
___A list of your plugins___
none

### Paper build number:
___This can be found by running `/version` on your server. `latest` is not a proper version number; we require the output of `/version` so we can properly track down the issue.___
This server is running Paper version git-Paper-86 (MC: 1.16.1) (Implementing API version 1.16.1-R0.1-SNAPSHOT)

### Anything else:
___Anything else you think may help us resolve the problem___

### PR DESCRIPTION
**Pull request by Machine-Maker**
Fixes #3930 
Slimes/Magma cubes did not pass on their Bukkit.Aware status to their "children" when they split.

### CODE REVIEW

File: Spigot-Server-Patches/0013-Allow-nerfed-mobs-to-jump-and-take-water-damage.patch [hunk 1]
```diff
@@ -3,6 +3,7 @@ From: Zach Brown <zach.brown@destroystokyo.com>
 Date: Tue, 1 Mar 2016 13:24:16 -0600
 Subject: [PATCH] Allow nerfed mobs to jump and take water damage
```
Comment (Proximyst) [inline Spigot-Server-Patches/0013-Allow-nerfed-mobs-to-jump-and-take-water-damage.patch]: Wrong patch for this; this isn't related, and should be its own patch.

**Review by Proximyst — CHANGES_REQUESTED** [review]

**Comment by Machine-Maker** [comment]
K, rebased.

**Review by Proximyst — DISMISSED** [review]

**Review by MiniDigger — DISMISSED** [review]

File: Spigot-Server-Patches/0601-Fix-nerfed-slime-when-splitting.patch [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+diff --git a/src/main/java/net/minecraft/server/EntitySlime.java b/src/main/java/net/minecraft/server/EntitySlime.java
+index e99fd88118a75f36cb93d02aa7c6029bcffd5f10..1422a927ad6ada9dfcb43be85d634f90ea5ff3b5 100644
+--- a/src/main/java/net/minecraft/server/EntitySlime.java
++++ b/src/main/java/net/minecraft/server/EntitySlime.java
+@@ -205,7 +205,7 @@ public class EntitySlime extends EntityInsentient implements IMonster {
+                 if (this.isPersistent()) {
+                     entityslime.setPersistent();
+                 }
+-
```
Comment (aikar) [inline Spigot-Server-Patches/0601-Fix-nerfed-slime-when-splitting.patch]: small nitpick but approving, can we get rid of the line removal

**Review by aikar — DISMISSED** [review]

File: Spigot-Server-Patches/0601-Fix-nerfed-slime-when-splitting.patch (hunk 2 above)
Comment (Machine-Maker) [reply Spigot-Server-Patches/0601-Fix-nerfed-slime-when-splitting.patch]: done

**Review by zachbr — DISMISSED** [review]

**Comment by MiniDigger** [comment]
@limbo-app ready

**Review by limbo-app — APPROVED** [review]
Pull request automatically approved for merging - a core team member has previously approved this pull request.
